packages/next/src/client/components/segment-cache/navigation.ts

```diff
@@ -218,6 +220,60 @@ export function navigate(
   }
 }
 
+export function refresh(
+  currentUrl: URL,
+  currentFlightRouterState: FlightRouterState,
+  currentNextUrl: string | null,
+  currentRenderedSearch: string,
+  currentCanonicalUrl: string,
+  currentCacheNode: CacheNode
+): SuccessfulNavigationResult | NoOpNavigationResult | MPANavigationResult {
+  // A refresh is a special case of a navigation where all the dynamic data
+  // on the current router is re-fetched. Most of the logic is handled within
+  // the ppr-navigations module. The main difference here is that we call
+  // startPPRRefresh instead of startPPRNavigation.
+  const now = Date.now()
+  const accumulation: NavigationRequestAccumulation = {
+    scrollableSegments: [],
+    separateRefreshUrls: null,
+  }
+  const task = startPPRRefresh(
+    now,
+    currentFlightRouterState,
+    currentNextUrl,
+    accumulation
+  )
+  if (task !== null) {
+    if (task.dynamicRequestTree !== null) {
+      listenForDynamicRequest(
+        currentUrl,
+        currentNextUrl,
+        task,
+        task.dynamicRequestTree,
+        null,
+        accumulation
+      )
+    }
+    const shouldScroll = false
```

Comment:
I think navigations default `shouldScroll` typically defaults to `true` unless explicitly overridden. Is there a rationale for it being `false` here?